gp-templates/error.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Template for errors.
+ */
+
+/** @var string $title */
+/** @var string $message */
+
+/* Translators: %s: title */
+gp_title( esc_html( sprintf( __( '%s &lt; GlotPress', 'glotpress' ), $title ) ) );
```

Comment:
If there is no title, it shows as `< GlotPress`.
```suggestion
gp_title( esc_html( sprintf( __( '%s &lt; GlotPress', 'glotpress' ), $title ? $title : __( 'Error', 'glotpress' ) ) );
```

Reply:
Done!